Documentation/disaster-recovery.md

```diff
@@ -106,3 +106,107 @@ kubectl create -f operator.yaml
+    4. Run `docker run -it --rm -v /var/lib/rook:/var/lib/rook ceph/ceph:v14.2.1-20190430 bash`. The Docker image tag should match the Ceph version used in the Rook cluster.
+
+        ```shell
+        container# cd /var/lib/rook
+        container# ceph-mon --extract-monmap m --mon-data ./mon-a/data  # Extract monmap from old ceph-mon db and save as m
+        container# monmaptool --print m  # Print the monmap content, which reflects the old cluster ceph-mon configuration.
+        container# monmaptool --rm a m  # Delete `a` from monmap.
+        container# monmaptool --rm b m  # Repeat this until all the old ceph-mons are removed
+        container# monmaptool --rm c m
+        container# monmaptool --rm d m
+        container# monmaptool --rm e m
+        container# monmaptool --add a 10.77.2.216:6789 m   # Replace it with the rook-ceph-mon-a address you got from previous command.
+        container# ceph-mon --inject-monmap m --mon-data ./mon-a/data  # Replace monmap in ceph-mon db with our modified version.
+        container# rm m
+        container# exit
+        ```
+
+1. Tell Rook to run as old cluster by running `kubectl edit secret/rook-ceph-mon` and changing `fsid` to the original `fsid`.
+1. Disable authentication by running `kubectl edit cm/rook-config-override` and adding content below:
+
+    ```
+    data:
+    config: |
+        [global]
+        auth cluster required = none
+        auth service required = none
+        auth client required = none
+        auth supported = none
+    ```
+
+1. Bring the Rook Ceph operator back online by running `kubectl edit deploy/rook-ceph-operator` and set `replicas` to 1.
+1. Watch the operator logs with `kubectl logs -f rook-ceph-operator-xxxxxxx`, and wait until the orchestration has settled.
+1. **STATE:** Now the new cluster should be up and running with authentication disabled.
+1. Run `kubectl exec -it rook-ceph-tools-XXXXXXX bash` to enter tools pod:
+
+    ```shell
+    tools# vi key
+    [paste keyring content saved before, preserving only `[client admin]` section]
+    tools# ceph auth import -i key
+    tools# rm key
+    ```
+
+1. Re-enable authentication by running `kubectl edit cm/rook-config-override` and removing auth configuration added in previous steps.
+1. Stop the Rook operator by running `kubectl edit deploy/rook-ceph-operator` and set `replicas` to 0.
+1. Shut down entire new cluster by running `kubectl delete deploy/X` where X is every deployment in namespace `rook-ceph`, except `rook-ceph-operator` and `rook-ceph-tools`, again. This time OSD daemons are present and should be removed too.
+1. Bring the Rook Ceph operator back online by running `kubectl edit deploy/rook-ceph-operator` and set `replicas` to 1.
+1. Watch the operator logs with `kubectl logs -f rook-ceph-operator-xxxxxxx`, and wait until the orchestration has settled.
+1. **STATE:** Now the new cluster should be up and running with authentication enabled.
```

Comment:
Following the **STATE:** trend from my above suggestions, it probably follows to add `ceph status` output here too.

Reply:
fixed